src/Reducers/initialState.js

```diff
@@ -37,7 +37,9 @@ const initialState = {
     selectedSearchResultCopy: [],
     selectedAll: false,
     employees: [],
-    selectedEmployees: []
+    selectedEmployees: [],
+    openModal: false,
```

Comment:
Does the redux state need to be aware of whether this modal is open or not? I feel that `openModal` could be more local state to the modal itself.